stable/memcached/values.yaml

```diff
@@ -61,6 +61,13 @@ tolerations: {}
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
 affinity: {}
 
+## Memcached pod Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+securityContext:
+  enabled: true
+  fsGroup: 11211
```

Comment:
Why these uid/guids? E.g 1001 is a more common value here.

Reply:
Only because it was used here -> https://github.com/docker-library/memcached/blob/master/alpine/Dockerfile#L4

Reply:
but I can put `1001` :)